src/library/Menu/themes.js

```diff
@@ -14,17 +13,11 @@ export const menuDividerTheme: MenuDividerThemeFn = (baseTheme) => ({
   ...baseTheme
 });
 
-export const menuGroupTheme: MenuGroupThemeFn = (baseTheme) => ({
-  MenuGroup_margin: baseTheme.space_stack_sm,
-
-  ...baseTheme
-});
-
 // [1] Deviation from h5 for optics relative to a MenuItem
-export const menuGroupTitleTheme: MenuGroupTitleThemeFn = (baseTheme) => ({
+export const menuGroupTheme: MenuGroupThemeFn = (baseTheme) => ({
   MenuGroupTitle_fontSize: baseTheme.fontSize_mouse, // [1]
   MenuGroupTitle_fontWeight: baseTheme.h5_fontWeight,
-  MenuGroupTitle_paddingTop: baseTheme.space_stack_md,
```

Comment:
Re: https://github.com/mineral-ui/mineral-ui/pull/796/commits/48d6ae8f0461e4dc26ac9f5b98f137d0779a05bb#r234349997

I don't think the change is this simple. While the space above the group title will sometimes be 24px (space_stack_lg), it won't always.